sdk/storage/azure-storage-blob-stress/src/main/java/com/azure/storage/blob/stress/utils/OriginalContent.java

```diff
@@ -116,4 +136,31 @@ private byte[] readHead(Path path) {
             throw LOGGER.logExceptionAsError(new UncheckedIOException(e));
         }
     }
+
+    public Flux<ByteBuffer> convertInputStreamToFluxByteBuffer(InputStream inputStream) {
```

Comment:
Let's not do this and make callers to `BinaryData.fromStream(inputStream).toFluxByteBuffer()` instead - this would work better for potentially huge contents